src/components/Dashboard/PositionsTable/PositionsTable.js

```diff
@@ -196,6 +198,17 @@ const PositionsTable = (props) => {
     excludeDataTableColumn,
   } = usePositionDataTableCompose(positionsFiltered, confirmAction);
 
+  /**
+   *
+   * @param {UserPositionsCollection} allPositions positions collection.
+   * @returns {Number} count of positions in last 24 hour.
+   */
+  const filterPastDayPositions = (allPositions) => {
+    const last24HoursTime = new Date().getTime() - 86400000;
```

Comment:
Instead of 86400000 perhaps 60*60*24*1000 makes it easier to understand where this number comes from.

Reply:
@cwagner22  Done.